### PR TITLE
[Queries] Added Text for Pinning

### PR DESCRIPTION
**Pull request by presstartgames**
^^

### CODE REVIEW

**Comment by presstartgames** [comment]
@mtrias, this adds some text for pinning on Queries Dashboard. Thoughts? 

I won't publish until VividCortex/ng-app#4352 is merged

File: content/how-to-use-vividcortex/queries.md [hunk 1]
```diff
@@ -33,6 +33,18 @@ The main area of the UI shows a high level overview of each query returned from
 * **First Seen** shows the date the query family was first seen by the application.
 * **Last Seen** shows the date the query family was last active.
 
+## Pinning
+
+Pinning allows you to save a small collection of queries, and show that collection at the top of the dashboard table. To pin a query, click the 'pin' icon to the left of the query. This will move the query to the top of the table, and will remain at the top even when paging or refreshing the browser.
```
Comment (mtrias) [inline content/how-to-use-vividcortex/queries.md]: maybe s / refreshing the browser / refreshing the page ?

File: content/how-to-use-vividcortex/queries.md [hunk 2]
```diff
@@ -33,6 +33,18 @@ The main area of the UI shows a high level overview of each query returned from
...
+
+![Queries](/img/docs/queries-dashboard-pinning.png)
+
+You can hide pinned queries by using the switch in the top right corner above the table.
+
+![Queries](/img/docs/pin-switch.png)
+
+To remove a query, click the colored icon to the left of the query. Or, you may remove every pinned query by clicking the link in the upper-right corner above the table.
```
Comment (mtrias) [inline content/how-to-use-vividcortex/queries.md]: Maybe s / To remove a query / To unpin a query ?

**Comment by mtrias** [comment]
Only nitpicks. Feel free to ignore. 🚢

**Comment by presstartgames** [comment]
Updated

**Pull request merged**